src/app/src/components/FilterSidebarFacilitiesTab.jsx

```diff
@@ -273,10 +273,10 @@ function FilterSidebarFacilitiesTab({
                                         oar_id: oarID,
                                     },
                                 }) => (
-                                    <Fragment key={oarID}>
+                                    <Fragment key={`${oarID}-${address}-${name}`}>
```

Comment:
> OAR IDs should already be unique, but this change also adds the name and address as parts of the React key for the rows shown in the infinite list.

Consider expanding this commit message to explain why we are adding values to the key if the OAR ID is already unique.

Reply:
I think I'll just drop this commit -- thinking about it again, it's not really doing anything here.